
 Make sure bundled version of squashfs_fs.h is used and not that of the build system. Build system version matches the version used by the build system kernel.
 Bundled version matches the version used by target system kernel (we ensure this by applying the required patches to the target kernel sources).

--- squashfs-tools/mksquashfs.c
+++ squashfs-tools/mksquashfs.c
@@ -62,7 +62,7 @@
 #include <sys/sysinfo.h>
 #endif
 
-#include <squashfs_fs.h>
+#include "squashfs_fs.h"
 #include "mksquashfs.h"
 #include "global.h"
 #include "sort.h"
--- squashfs-tools/read_fs.c
+++ squashfs-tools/read_fs.c
@@ -43,7 +43,7 @@
 #include <endian.h>
 #endif
 
-#include <squashfs_fs.h>
+#include "squashfs_fs.h"
 #include "read_fs.h"
 #include "global.h"
 
--- squashfs-tools/sort.c
+++ squashfs-tools/sort.c
@@ -35,7 +35,7 @@
 #include <stdlib.h>
 #include <stdint.h>
 
-#include <squashfs_fs.h>
+#include "squashfs_fs.h"
 #include "global.h"
 #include "sort.h"
 
